app/views/handlers/list_edit_question.py

```diff
@@ -13,6 +13,15 @@ def is_location_valid(self):
             return False
         return True
 
+    def get_next_location_url(self):
+        if self._is_returning_to_section_summary():
+            return self.get_section_summary_url()
+
+        if url := self._get_first_incomplete_repeating_block_url():
+            return url
```

Comment:
I'd put it in repeating question but I also need this in `ListEditQuestion` because if you are on the edit block for a list collector with incomplete repeating blocks, you also need to jump to the url of the first incomplete one, and both inherit from ListAction so I think thats where it needs to go

Reply:
You might be able to have `ListRepeatingQuestion` inherit `ListEditQuestion` - I originally had it doing this in the Looping 2.5 impl but there was were calls preventing compatibility in `ListAction` `handle_post` which was called by `ListEditQuestion`. 
However this was removed in[ this PR](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1149). 
This means you could call `ListEditQuestion.handle_post` safely from `ListRepeatingQuestion.handle_post` now. They also share identical `get_next_location_url`.
So this can then be declared only in ListEditQuestion and remove some duplicated code.

Reply:
yes! that will totally do it

Reply:
Done 👍  and removed the method as Pete suggests since its back to only being used once, thanks both

Reply:
Looks good to me, consider my comments on this thread resoved 👍